kotlin/context-menu-extension/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Are files from .idea added intentionally?

Reply:
I had the same question :) @DarthWeirdo wrote in DM that his IDEA was set up to use JDK 8 for Gradle, while our SDK is compiled to be run on JDK 11 or newer. And he had an error on runtime. I don't think it's a good reason to include any .idea file here.

Reply:
removed it